wordy/base/worddict.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import annotations
+from typing import List
+
+class WordDict():
+    """To create a new WordDict use either WordDict.fromFile() or WordDict.fromList()"""
+    def __init__(self):
+        self.words = []
+    
+    @classmethod
+    def fromFile(cls, fileName: str) -> WordDict:
```

Comment:
Can we stay consistent and use `snake_case` for everything in the project? The class names are fine but everything else in Python should use `snake_case`. Pycharm should be able to rename each variable pain-free if you highlight, then click refactor at the top > rename. There's a keyboard shortcut too.